src/Concordium/Client/Runner.hs

```diff
@@ -3870,10 +3849,10 @@ processTransaction ::
   => BSL.ByteString
   -> Int
   -> ClientMonad m Types.BareBlockItem
-processTransaction source networkId =
+processTransaction source _networkId =
   case AE.eitherDecode source of
     Left err -> fail $ "Error decoding JSON: " ++ err
-    Right t  -> processTransaction_ t networkId True
+    Right t  -> processTransaction_ t _networkId True
```

Comment:
Why did you make this change?

Reply:
Ok, I also saw `_networkId` changes in one or more of the other PRs.

Reply:
Why are we not just removing the network ID? It is not used in V2 at all.

Reply:
I added a TODO on Zenhub for this.